index.html

```diff
@@ -280,28 +222,63 @@ <h4 class="frame-name">CSS</h4>
             <li class="tabs">
               <h4 class="tab-title">Framework</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
           <ul class="pro-languages separator-bottom">
             <li class="tabs">
               <h4 class="tab-title">Skills</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
         </div>
       </div>
     </section>
+    <section class="form">
```

Comment:
There's still some room to improve the design, here are some things that you could do in order to stick as much as possible to the FIGMA template:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/212958588-6785d0d9-f8da-486f-91e3-1c909101bbce.png) | ![image](https://user-images.githubusercontent.com/77069011/212958482-e476b04c-3878-4df1-8432-dbc3357639a5.png) |


- [ ] 🚧 Consider fixing the `border-radius` at the top-left corner, you could use `border-top-left-radius: 6rem`, which I personally think is a good approximation, instead of percentages. Feel free to use another solution! 🔧  

- [ ] 🚧 Please consider fixing the path to the background image, it is not displaying correctly on my end. Try using a relative path.  🔧 

- [ ] 🚧 Please consider adding some horizontal spacing to the paragraph, and centering the text as the Figma template suggests. 🔧 

- [ ] 🚧 Please add some padding to each input field and the button, the text is so close to the edges. 🔧 

- [ ] 🚧 Please add some margin at the end, after the button, it is touching the end of the screen, which doesn't look that good regarding the design. 🔧 

Each one of these changes will improve tremendously the look and feel of your page, so it is worth spending some time applying them! 🤓 🛠️